src/decision/src/LidarDecision.cpp

```diff
@@ -31,6 +32,143 @@ void LidarDecision::scanCallBack(const sensor_msgs::LaserScan::ConstPtr& raw_sca
     // Deal with new messages here
 }
 
+
+
+
+
+double pi = 3.1415926;
+
+
+bool LidarDecision::obstacle_in_range(double min_angle, double max_angle, float obstacle_distance,
+                                      const sensor_msgs::LaserScan::ConstPtr& raw_scan) {
+    int i;
+    int init_pos_vec, end_pos_vec;
+
+    init_pos_vec = static_cast<int>((min_angle - raw_scan->angle_min) / raw_scan->angle_increment);
+    end_pos_vec = static_cast<int>((max_angle - raw_scan->angle_min) / raw_scan->angle_increment);
+
+    /*bool exist_obstacle = false;
```

Comment:
Please delete any commented out code, it helps keep everything clean and readable. (Don't worry about losing it, that's what Git is for!)